data-raw/healthy-lives/cardiovascular-conditions.R

```diff
@@ -0,0 +1,52 @@
+# ---- Load packages ----
+library(tidyverse)
+library(rio)
+library(geographr)
+library(demographr)
+
+# ---- Get data ----
+# Population
+population_2022 <- population22_ltla19_scotland |>
+  filter(sex == "Persons") |>
+  select(
+    ltla19_name,
+    ltla19_code,
+    population_2022 = all_ages
+  )
+
+# Cardiovascular Conditions
+# Source: https://www.opendata.nhs.scot/dataset/scottish-heart-disease-statistics/resource/5379a655-d677-46cf-814d-bc0574ac59e4
+cardiovascular_conditions_raw <- import("https://www.opendata.nhs.scot/dataset/0e17f3fc-9429-48aa-b1ba-2b7e55688253/resource/5379a655-d677-46cf-814d-bc0574ac59e4/download/hd_activitybyca.csv")
+
+# ---- Clean data ----
+cardiovascular_conditions <- cardiovascular_conditions_raw |>
+  filter(`FinancialYear` == "2022/23",
+         `AgeGroup` == "All",
+         `AdmissionType` == "All",
+         `Sex` == "All") |>
+  rename(ltla19_code = 2,
+         discharge_number = 11) |>
+  group_by(ltla19_code) |>
+  summarise(
+    total_discharge_number = sum(discharge_number, na.rm = TRUE)
+  ) |>
+  mutate(year = "2022/23",
+         av_discharge_number = (total_discharge_number / 4))|>
```

Comment:
cos this was the total for across 4 health conditions so to get an average I divided by 4, but seems upon reflection this may have been an unnecessary step...